ansible/configs/kni-osp/files/repos_template.j2

```diff
@@ -0,0 +1,44 @@
+[rhel-8-server-rpms]
```

Comment:
You might want to consider using the Satellite repo method. I don't believe we host any rhel 8 repos that can be accessed via the file method, but I am not 100% on this.

Reply:
Can you please point me to that method ? I need several repos that I'm not even sure are available on sat ?

Reply:
set-repositories role takes repo_method: satellite 
https://github.com/redhat-cop/agnosticd/tree/development/ansible/roles/set-repositories
I am contacting ops to have them create the content view with the repositories listed in this j2 for you. See email sent.